collector/src/compile/benchmark/mod.rs

```diff
@@ -504,17 +500,19 @@ pub fn get_compile_benchmarks(
     }
 
     // All prefixes/suffixes must be used at least once. This is to catch typos.
-    let check_for_unused = |option, substrings: Option<HashMap<&str, usize>>| {
-        if let Some(substrings) = substrings {
+    let check_for_unused = |option, substrings: HashMap<&str, usize>| {
+        if !substrings.is_empty() {
             let unused: Vec<_> = substrings
                 .into_iter()
                 .filter_map(|(i, n)| if n == 0 { Some(i) } else { None })
                 .collect();
             if !unused.is_empty() {
                 bail!(
-                    "Warning: one or more unused --{} entries: {:?}",
+                    "Warning: one or more unused --{} entries: {:?} found.\n\
```

Comment:
Could you please use a multiline string here? (`r#"..."#`). The trailing backslash looks quite weird.

Reply:
Done.